_posts/2019-12-02-hhvm-4.34.markdown

```diff
@@ -0,0 +1,29 @@
+---
+title: "HHVM 4.34"
+layout: post
+author: jjergus
+category: blog
+---
+
+HHVM 4.34 is released! This release marks the end of support for 4.28;
+4.29&ndash;4.33 remain supported, as does the 4.8 LTS release.
+
+# Highlights
+
+- Fixes bugs in `hh_client --concatenate-all`. This command should now provide a
```

Comment:
This previously hasn't been documented at all due to buginess, so should probably be a bit more in-depth here, and describe the feature. Maybe:

fixed a bug in `hh_client --concatenate-all`; this is a new feature, providing a reliable way to produce a single-file version of a Hack project, respecting dependency order, and resolving mixed namespaces and conflicting `use` statements